recipes/mailio/config.yml

```diff
@@ -1,4 +1,6 @@
 versions:
+  "0.25.2":
+    folder: all
```

Comment:
We also need to remove the lines below, as we have removed those versions from the conandata.yml file as well. I missed that change 😅